### PR TITLE
finish exercise

### PR DESCRIPTION
**Pull request by devtanole**
Screenshot 2024-07-14 at 2 02 09 PM

### CODE REVIEW

**Review by hesam — CHANGES_REQUESTED** [review]
This is good work! If possible, see if you could improve (generalize) it by taking advantage of arrays! Rather than product1 product2 properties, how about using a property called products which stores any number of items which happen to be inside an order... You'll have to define a second new interface to represent each product.

**Comment by devtanole** [comment]
Screenshot 2024-07-15 at 12 59 01 AM

File: typescript-data-model/order-history.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
+// interface product {prodArr: [{prodName: string; author?: string; prodTotal: number;}];}
```
Comment (hesam) [inline typescript-data-model/order-history.ts:1]: interface product {prodName: string; author?: string; prodTotal: number;}

File: typescript-data-model/order-history.ts [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+interface order {
+  orderPlaced: string;
+  orderTotal: number;
+  orderNumber: string;
+  shipTo: string;
+  delivery: string;
+  returnWindow: string;
+  prodArr: any[];
```
Comment (hesam) [inline typescript-data-model/order-history.ts:9]: prodArr: product[];

**Review by hesam — CHANGES_REQUESTED** [review]
Almost there. I left 2 comments inside your code to fix...

**Review by hesam — APPROVED** [review]
Nice. Be sure to include the changes I commented earlier. 👍

**Pull request merged**